images/openstack/Dockerfile.ci

```diff
@@ -15,7 +15,7 @@ COPY --from=builder /go/src/github.com/openshift/installer/images/openstack/rdo-
 
 # Install Dependendencies for tests
 # https://github.com/openshift/origin/blob/6114cbc507bf18890f009f16ee424a62007bc390/images/tests/Dockerfile.rhel
-RUN yum install --setopt=tsflags=nodocs -y git gzip util-linux && yum clean all && rm -rf /var/cache/yum/* && \
+RUN yum install --setopt=tsflags=nodocs -y git gzip util-linux && \
```

Comment:
The reason why we clean the yum cache is because we want to keep each layer the smallest and how container image layering works. The RUN instruction is basically a layer and if you don't clean the yum cache in the same instruction it makes you container image unnecessarily fat.